bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/NetatmoModuleHandler.java

```diff
@@ -100,6 +102,9 @@ protected void updateChannels(Object module) {
         }
     }
 
+    public void updateMeasurements() {
+    }
+
```

Comment:
Is it still required now that you declared this method in AbstractNetatmoThingHandler ?

Reply:
Yes, its needed in NAMainHandler to tell it's modules to update.  It could be moved back to NetatmoModuleHandler, but shouldn't hurt to be here.